app/controllers/MaillotController.java

```diff
@@ -0,0 +1,49 @@
+package controllers;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import play.mvc.Controller;
+import play.mvc.Result;
+import repository.interfaces.MaillotRepository;
+import repository.interfaces.NotificationRepository;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'repository.interfaces.NotificationRepository'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)